nbbuild/licenses/CC-BY-4.0

```diff
@@ -0,0 +1,429 @@
+Attribution-ShareAlike 4.0 International
```

Comment:
This is `CC-BY-SA-4.0`. Given the wording of the license in the attribution clause, this needs an entry in the notice section.

Reply:
At the end I've removed the CC-BY-SA-4.0